platform/linux-generic/arch/aarch64/odp_crypto_armv8.c

```diff
@@ -239,28 +242,34 @@ odp_crypto_alg_err_t aes_gcm_encrypt(odp_packet_t pkt,
 	uint8_t *data = odp_packet_offset(pkt, in_pos, &seg_len, NULL);
 
 	if (odp_unlikely(odp_packet_is_segmented(pkt)) ||
```

Comment:
Might have been nice to get this and the if-esle block at line 269 to separate functions as they seem to repeat in encrypt and decrypt, but I guess this becomes a bit cumbersome due to the tag handling asymmetry introduced in a following patch.